kolibri/content/api.py

```diff
@@ -185,6 +188,26 @@ class OptionalPageNumberPagination(pagination.PageNumberPagination):
     page_size_query_param = "page_size"
 
 
+class AllContentCursorPagination(pagination.CursorPagination):
+    page_size = 10
```

Comment:
This is arbitrary and can be changed if we think we would show more by default - @khangmach @jtamiace @christianmemije please feel free to weigh in.